### PR TITLE
Update the readme.

### PR DESCRIPTION
**Pull request by reiterl**
Update the readme for the media service OS4.

### CODE REVIEW

**Comment by reiterl** [comment]
@jsangmeister  or @GabrielInTheWorld  can you pls review this, because we need to change the `Readme`.

File: README.md [hunk 1]
```diff
@@ -1,53 +1,33 @@
...
+"MEDIA_DATABASE_USER_FILE": Path to the (secret) file, which contains the
+username,
+"MEDIA_DATABASE_PASSWORD_FILE": Path to the (secret) file, which contains the
+password,
+"BLOCK_SIZE": Default 4096. The size of the blocks, the file is chunked into.
+              4096 seems to be a good default,
+"PRESENTER_HOST": Host of the presenter service,
+"PRESENTER_PORT": Port of the presenter service,
+```
+all config must be set.
```
Comment (jsangmeister) [inline README.md:22]: No, they all have defaults. Please provide them here

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Comment by jsangmeister** [comment]
ping @reiterl

**Comment by reiterl** [comment]
Rebased and added the defaults to the readme.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**